processes/security_team_members.md

```diff
@@ -0,0 +1,92 @@
+# Node.js Security Team
+
+Node.js security team members are expected to keep all information that they have
+privileged access to by being on the team completely private to the team. This
+includes notifying anyone outside the team of issues that have not yet been
+disclosed publicly, including the existence of issues, expectations of upcoming
+releases, and patching of any issues other than in the process of their work as
+a member of the security team.
+
+Membership on the security teams can be requested via an issue in the TSC repo,
+and must be approved by current team members.
+
+Members of the security teams should indicate that they accept the privacy
+policies by PRing their acceptance to this file.
+
+## Team that triages security reports against node core
+
+- @bnoordhuis - **Ben Noordhuis**
+- @indutny - **Fedor Indutny**
+- @rvagg - **Rod Vagg**
+- @jasnell - **James M Snell**
+- @shigeki - **Shigeki Ohtsu**
+- @MylesBorins - **Myles Borins**
+
+List is from ["security" alias](https://github.com/nodejs/email/blob/master/iojs.org/aliases.json).
+
+## Team with access to security issues
+
+- @ChALkeR - **Сковорода Никита Андреевич**
+- @Fishrock123 - **Jeremiah Senkpiel**
+- @MylesBorins - **Myles Borins**
+- @Trott - **Rich Trott**
+- @addaleax - **Anna Henningsen**
+- @bnoordhuis - **Ben Noordhuis**
+- @cjihrig - **Colin Ihrig**
+- @dougwilson - **Douglas Wilson**
+- @ejratl - **Emily Ratliff**
+- @evanlucas - **Evan Lucas**
+- @evilpacket - **Adam Baldwin**
+- @grnd - **Danny Grander**
+- @indutny - **Fedor Indutny**
+- @jasnell - **James M Snell**
+- @jbergstroem - **Johan Bergström**
+- @joaocgreis - **João Reis**
+- @joshgav - **Josh Gavant**
+- @mhdawson - **Michael Dawson**
+- @mscdex - **Brian White**
+- @ofrobots - **Ali Ijaz Sheikh**
+- @rvagg - **Rod Vagg**
+- @saghul - **Saúl Ibarra Corretgé**
+- @sam-github - **Sam Roberts**
+- @shigeki - **Shigeki Ohtsu**
+- @targos - **Michaël Zasso**
+- @thefourtheye - **Sakthipriyan Vairamani**
+- @trevnorris - **Trevor Norris**
+
+List is from [nodejs/teams/security](https://github.com/orgs/nodejs/teams/security/members).
+
+## Team with access to private security patches
+
```

Comment:
So there need to be explicitly defined groups here:

We have:

1. The Triage Team -- the small group that looks at initial reports and determines who the fix team should be.
2. The Fix Team -- the small group of Collaborators with access to the private repo for fixing a specific issue
3. The Review Team -- the slight larger group of trusted Collaborators who review and validate a fix immediately before release.

The group below should essentially be the Review Team. They may not be the ones actually working on the fix, and may not get the full details of the vulnerability right away, but they are expected to review and sign off on the fix before it goes out the door. Personally, I think the Review Team should be limited strictly to TSC members. If non-TSC members are required to help resolve an issue, they should be brought in on the Fix Team.

Reply:
In WG meetings, we felt that proposals to change the current process for core would best be raised after #54 is done, so changes to the core process can be proposed in the context of the tooling to be used. Just a suggestion, of course the TSC can change the sec process and team arrangement at any time. If you would like to kick off that discussion immediately I suggest opening a separate issue or PR.